libraries/AP_Motors/AP_MotorsHeli_Single.cpp

```diff
@@ -116,6 +116,13 @@ const AP_Param::GroupInfo AP_MotorsHeli_Single::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("GYR_GAIN_ACRO", 11, AP_MotorsHeli_Single,  _ext_gyro_gain_acro, 0),
 
+    // @Param: COL_CTRL_DIR
+    // @DisplayName: Collective Control Direction
+    // @Description: Collective Control Direction - 0 for Normal. 1 for Reversed
+    // @Values: 0: Normal, 1: Reversed
+    // @User: Standard
+    AP_GROUPINFO("COL_CTRL_DIR", 19, AP_MotorsHeli_Single, _collective_direction, AP_MOTORS_HELI_SINGLE_COLLECTIVE_DIRECTION_NORMAL),
+
     // Indices 16-18 were used by RSC_PWM_MIN, RSC_PWM_MAX and RSC_PWM_REV and should not be used
```

Comment:
This is a bit of a nit but this comment should be in front of the COL_CTRL_DIR parameter to keep the numbering in sequence.